pkg/scaffold/olm-catalog/csv.go

```diff
@@ -209,7 +209,7 @@ func (s *CSV) initCSVFields(csv *olmapiv1alpha1.ClusterServiceVersion) {
 	csv.Spec.Version = *semver.New(s.CSVVersion)
 	csv.Spec.DisplayName = getDisplayName(s.ProjectName)
 	csv.Spec.Description = "Placeholder description"
-	csv.Spec.Maturity = "alpha"
+	csv.Spec.Maturity = "Basic Install"
```

Comment:
Is the term "Basic Install" documented anywhere or standardized, as to me it doesn't convey that much?

Reply:
yes, we link to it in the csv-generation.md and in building-your-csv.md on OLM docs. It's basically describing the capability of an operator, i.e. what parts of an application lifecycle it's able to automate.